content/chainguard/chainguard-images/how-to-use/use-with-openshift.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "How to Use Chainguard Containers with OpenShift"
+linktitle: "Using With OpenShift"
+type: "article"
+description: "Special considerations for using Chainguard Containers with OpenShift"
+date: 2025-06-17T08:49:31+00:00
+lastmod: 2025-06-17T08:49:31+00:00
+draft: false
+tags: ["Chainguard Containers", "OpenShift", "Product"]
+images: []
+menu:
+  docs:
+    parent: "chainguard-images"
+weight: 008
+toc: true
+---
+
+When [Using Chainguard Containers](./how-to-use-chainguard-images.md) with OpenShift, there are some adjustments that need to be made to the usual process.
+
+# Adjust Ownership and Permissions
+
+By default, OpenShift Container Platform runs containers using an arbitrarily assigned User ID (UID), as described in the [Red Hat documentation](https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/images/creating-images#use-uid_create-images).
+
+There are required access settings for an image to support running as an arbitrary user:
+
+- Directories and files that are written to by processes in the image must be owned by the `root` group and that group must have both `read` and `write` permissions
+- Files that will be executed must also have group execute permissions
+
+Following the Red Hat requirements, to use Chainguard Containers you must make the following change in your Dockerfile to set the required ownership and permissions:
+
+```
+RUN chgrp -R 0 /some/directory && \
```

Comment:
Can you introduce the example directories / path structure and then use those as the example info.